src/nni_manager/training_service/pai/paiTrainingService.ts

```diff
@@ -52,6 +52,7 @@ abstract class PAITrainingService implements TrainingService {
     protected authFileHdfsPath: string | undefined = undefined;
     protected portList?: string | undefined;
     protected paiJobRestServer?: PAIJobRestServer;
+    protected protocol: string = 'http://';
```

Comment:
protocol should be `http` instead of `http://`

Reply:
fixed